tests/profiling/collector/test_stack.py

```diff
@@ -627,7 +627,7 @@ def _nothing():
                 main_thread_found = True
         elif event.task_id in {t.ident for t in threads}:
             # Make sure we capture the sleep call and not a gevent hub frame
-            assert event.frames[0][2] in ("_nothing", "sleep", "get_ident", "__bootstrap_inner")
+            assert event.frames[0][2] in ("_nothing", "sleep", "get_ident", "__bootstrap_inner", "switch")
```

Comment:
```suggestion
            assert event.frames[0][2] in ("_nothing", "sleep", "get_ident", "__bootstrap_inner", "switch", "run")
```
https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/12132/workflows/f69d2b6e-e622-4fea-aacd-c41b6d060432/jobs/892887